modules/nback.llvm/llvm.bal

```diff
@@ -26,14 +26,14 @@ public readonly distinct class Value {
     }
 }
 
-public function constInt(IntType ty, int val) returns Value {
```

Comment:
Remove default and change argument order back to match LLVMConstInt

Reply:
Fixed